config.go

```diff
@@ -95,7 +96,8 @@ type config struct {
 	MaxGenTime            time.Duration `long:"maxgentime" ini-name:"maxgentime" description:"The share creation target time for the pool. Valid time units are {s,m,h}. Minimum 2 seconds. This currently should be below 30 seconds to increase the likelihood a work submission for clients between new work distributions by the pool."`
 	PaymentMethod         string        `long:"paymentmethod" ini-name:"paymentmethod" description:"The payment method of the pool. {pps, pplns}"`
 	LastNPeriod           time.Duration `long:"lastnperiod" ini-name:"lastnperiod" description:"The time period of interest when using PPLNS payment scheme. Valid time units are {s,m,h}. Minimum 60 seconds."`
-	WalletPass            string        `long:"walletpass" ini-name:"walletpass" description:"The wallet passphrase."`
+	WalletPass            string        `long:"walletpass" ini-name:"walletpass" description:"The wallet passphrase to use when paying dividends to pool contributors."`
+	WalletAccount         uint32        `long:"walletaccount" ini-name:"walletaccount" description:"The wallet account to use when paying dividends to pool contributors."`
```

Comment:
The description should read "The wallet account that will receive mining rewards, when not mining in solo pool mode".